cfe_internal/update/update_policy.cf

```diff
@@ -243,16 +243,6 @@ bundle agent cfe_internal_update_policy_cpv
       action => u_immediate,
       classes => u_if_repaired("validated_updates_ready");
 
-    any::
```

Comment:
instead of `any::` you can just do `validated_updates_ready|(enterprise_edition.policy_server)::

Reply:
How about using an existing class guard below? ```am_policy_hub|validated_updates_ready::  # policy hub should always put masterfiles in inputs in order to check new policy```